scripts/setup.sh

```diff
@@ -26,11 +26,12 @@ if [ ! "$(ls -A /data)" ]; then
     curl -L -o /home/pingu/msql.deb https://dev.mysql.com/get/Downloads/Connector-J/mysql-connector-java_8.0.26-1ubuntu20.04_all.deb
     apt install -y /home/pingu/msql.deb
 
-    # Set up rcon
-    curl -L -o /home/pingu/rcon.tar.gz https://github.com/itzg/rcon-cli/releases/download/1.4.8/rcon-cli_1.4.8_linux_amd64.tar.gz
-    tar -xf /home/pingu/rcon.tar.gz -C /home/pingu/
 fi
 
+# Set up rcon
+curl -L -o /home/pingu/rcon.tar.gz https://github.com/itzg/rcon-cli/releases/download/1.4.8/rcon-cli_1.4.8_linux_amd64.tar.gz
+tar -xf /home/pingu/rcon.tar.gz -C /home/pingu/
+
```

Comment:
What's the problem with having it in the if-statement?
It should trigger on first startup only, otherwise we download it every time 
... I guess with the -L flag it's not the end of the world, but in that case we should perhaps clean up the tar command?

Reply:
Problem is the home directory seems to get wiped if you restart the service
Martin and I had some problems with it earlier

How could we clean up the tar command?

Reply:
after discussion on discord it is deemed fine